backend/src/server/routes/v1/dashboard-router.ts

```diff
@@ -633,4 +634,182 @@ export const registerDashboardRouter = async (server: FastifyZodProvider) => {
       };
     }
   });
+
+  server.route({
+    method: "GET",
+    url: "/secrets-quick-search",
```

Comment:
Feels a bit weird by this name. I think this should be something that denotes a deep search meaning.

Let's raise this against team.

Reply:
I think changing to `secrets-deep-search` should suffice?